src/main/java/md/maib/entity/Customer.java

```diff
@@ -0,0 +1,130 @@
+package md.maib.entity;
+
+import jakarta.persistence.*;
+import java.util.List;
+import java.util.Objects;
+
+@Entity
+@Table(name = "users")
+public class Customer {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "user_id")
+    private Long id;
+
+    @Column(name = "cvv", nullable = false)
+    private String cvv;
+
+    @Column(name = "first_name", nullable = false)
+    private String firstName;
+
+    @Column(name = "last_name", nullable = false)
+    private String lastName;
+
+    @Column(name = "pan", nullable = false, unique = true, length = 16)
+    private String pan;
+
+    @OneToMany(mappedBy = "customer", cascade = CascadeType.ALL, fetch = FetchType.EAGER)
```

Comment:
Read about **FetchType**. 
Eager have to be used very carefully, doesn't see any need for that.